src/components/files/AutoSuggestor.vue

```diff
@@ -75,44 +77,53 @@ export default {
       return suggestion.item.hierarchyFlat + " : " + suggestion.item.fullName;
     },
     async fetchResults(input) {
-      if (this.isExternal) {
-        return this.fetchExternal(input);
-      } else {
-        const res = await this.fetchInternal(input);
-        return res;
-      }
-    },
-    submitSelected() {
-      if (!this.selected) return;
-      this.$emit("select", { value: this.selected });
-    },
-    async fetchExternal(input) {
       if (input.length < minAutoComplete) {
         this.suggestions = [];
         return [];
       }
-      await delegatorsApi.searchUserByName(input);
+      this.input = input;
+      return await this.debounce();
+    },
+    async fetchExternal() {
 
-      const res = await delegatorsApi.searchUserByName(input);
+      const res = await delegatorsApi.searchUserByName(this.input);
       const users = res.data.users;
       if (users) {
         this.suggestions = [{ data: users }];
       }
       return users ? users : [];
     },
-    async fetchInternal(input) {
-      if (input.length < minAutoComplete) {
-        return [];
-      }
-      const res = await usersApi.searchUserByName(input);
+    async fetchInternal() {
+      const res = await usersApi.searchUserByName(this.input);
       const users = res.data.users;
       if (users) {
         this.suggestions = [{ data: users }];
       }
       return users ? users : [];
+    },
+    submitSelected() {
+      if (!this.selected) return;
+      this.$emit("select", { value: this.selected });
```

Comment:
Clear the search-bar input after this event

Reply:
done.